target/linux/ath79/dts/ar9344_mikrotik_routerboard-sxt-5nd-r2.dts

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar9344_mikrotik_sxt5n.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-sxt-5nd-r2", "qca,ar9344";
```

Comment:
"routerboard" seems to be not used for this device, so I'd go with
mikrotik,sxt-lite5 or mikrotik,sxt-5nd-r2 here.

Reply:
This can be checked via sysfs. E.g. on my SXTsq 2:

```
# cat /sys/firmware/mikrotik/hard_config/board_product_code 
RouterBOARD SXTsq 2nD
```

Reply:
Although it's also true that MikroTik has been very inconsistent in their use of "RouterBOARD" prefix in product naming. For the exact same device from one model to another it appears to come and go at random. I would humbly suggest that we stick to using it, for consistency's sake across their RB range.

Reply:
I'd consider calling yours mikrotik,sxtsq-lite2 based on the webpage/brochure ...

Reply:
I guess you can s/routerboard// globally then :)
All these devices were at one point or the other "RouterBOARDs" (as still hinted by the RB product prefix), and now it seems MikroTik is phasing out that naming. Until it comes back again and then we can have this discussion all over again ;)

Reply:
Yes, there is arguments for adding it everywhere and arguments for removing it everywhere. Just looked at the overview here:
https://mikrotik.com/products

Funnily, they provide a category RouterBOARD on the left which only contains the real boards without a box.

However, many of the boxed routers and switches contain the RB prefix in their marketed name. On the other hand, there is stuff like CCR1009-7G-1C-PC in the same category that doesn't even use RB in the technical names.

I will give it some more thinking ...

Reply:
probably we can find a common naming-scheme. I just opened #3270 for discussion / change.